@stellar/lyra-api/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@stellar/lyra-api",
+  "main": "build/index.js",
+  "types": "build/@stellar/lyra-sdk/src/index.d.ts",
+  "prettier": "@stellar/prettier-config",
+  "version": "1.0.0-alpha",
+  "description": "Utility functions to interact with Lyra extension",
+  "repository": {
+    "url": "git@github.com:stellar/lyra.git",
+    "type": "git"
+  },
+  "author": "Stellar Development Foundation <hello@stellar.org>",
+  "browserslist": {
+    "production": [
+      "last 3 chrome version",
+      "last 3 firefox version"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version"
+    ]
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@lyra/api": "1.0.0",
+    "typescript": "~3.7.2"
```

Comment:
Is TS needed as a dep for production? At first glance it seems like it'd be a devdep